### PR TITLE
ENT-5879: Fixed log level for failure to stat remote files without missing_ok (3.12)

### PR DESCRIPTION
**Pull request by nickanderson**
In versions before 3.12.0 it was not possible to suppress errors generated from
promises to copy remote files that did not exist. Because of this limitation,
this specific log message was moved from ERROR to INFO as it could generate many
logs that were impossible to avoid, whereas INFO log messages are not always
emitted, especially by regularly scheduled runs.

Version 3.12.0 introduced the missing_ok attribute which allows for clients to
avoid considering a missing remote file as an error.

This change restores the log level for the message to ERROR. Users should
instrument their policy with missing_ok if they are promising copies of files
that may not exist.

Ticket: ENT-5879
Changelog: Restored log level for failure to state remote files without missing_ok to ERROR
(cherry picked from commit 1d09f0da4f7c1180fde1d1568e4c8be9a73729b1)

### CODE REVIEW

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/6311/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/6311/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-6311/

**Comment by nickanderson** [comment]
I do not believe that test failure is related to this change. @cf-bottom try jenkins again

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/6321/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/6321/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-6321/

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please since i dont see a green build from there for this PR yet.

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/6334/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/6334/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-6334/

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/6357/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/6357/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-6357/

**Review by olehermanse — APPROVED** [review]
Looks good to me, just make sure it doesn't cause any 🟥

**Comment by nickanderson** [comment]
> Looks good to me, just make sure it doesn't cause any red_square

Yeah, thats what been trying to get clear from for days, there has been redness, but redness NIH,

**Review by vpodzime — APPROVED** [review]
ACK

**Comment by vpodzime** [comment]
sequential tests retry with cfengine/system-testing#381 merged: [Build Status](https://ci.cfengine.com/job/sequential-tests/1781/)

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/6365/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/6365/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-6365/

**Review by craigcomstock — APPROVED** [review]
wait for green build, should work after vrata merged my fix for 3.12.x CI

**Comment by nickanderson** [comment]
> wait for green build, should work after vrata merged my fix for 3.12.x CI

so much waiting for these builds

**Comment by nickanderson** [comment]
> > wait for green build, should work after vrata merged my fix for 3.12.x CI
> 
> so much waiting for these builds

Started 5 hr 10 min ago
Build has been executing for 5 hr 10 min on master

**Comment by nickanderson** [comment]
ZOMG, finally green 

image

**Pull request merged**
